core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -1206,6 +1236,563 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
+
+                                        UNWIND_CSP(relative_depth,
+                                                   LABEL_TYPE_CATCH);
+#if 0
+                                                /* unwind to try frame 
+                                                 * the relative depth may be 0 if the delegate is on a try, but maybe larger 
+                                                 * if there are additional blocks in the try block or 
+                                                 * if THAT DELEGATE here is found by another delegate or a rethrow */
+                                                frame_csp -= relative_depth;
+
+                                                /* drop handlers and values pushd in try block */
+                                                frame_sp = (frame_csp-1)->frame_sp;
+
+                                                /* convert the discovered try frame into catch frame, this frame cannot
+                                                    * longer seen as a try frame, because the handlers are not longer there
+                                                    */
+                                                (frame_csp-1)->label_type = LABEL_TYPE_CATCH;
+#endif
+
+                                        /* leave the block (the delegate is
+                                         * technically not inside the frame) */
+                                        frame_csp--;
+
+                                        /* unwind to delegated frame */
+                                        frame_csp -= lookup_depth;
+
+                                        /* transfer exception values */
+                                        uint8 tag_type_index =
+                                            module->module
+                                                ->tags[exception_tag_index]
+                                                .type;
+                                        uint32 cell_num_to_copy =
+                                            wasm_types[tag_type_index]
+                                                ->param_cell_num;
+                                        /* push exception values for catch */
+                                        if (cell_num_to_copy > 0) {
+                                            word_copy(frame_sp,
+                                                      frame_sp_old
+                                                          - cell_num_to_copy,
+                                                      cell_num_to_copy);
+                                        }
+                                        frame_sp += cell_num_to_copy;
+
+                                        /* tag_index is already stored in
+                                         * exception_tag_index */
+                                        goto find_a_catch_handler;
+                                    } break;
```

Comment:
No need to add `break;`, seems it is dead code.

Reply:
Updated accordingly.